test/packages/bad_knowledge_base/kibana/kb_entry/foo/content.0.json

```diff
@@ -0,0 +1,106 @@
+{
+  "documents": [
```

Comment:
I wonder if this file could be just a ndjson file.

Is the `documents` entry needed? Do we expect other entries in this file?

How are these files generated? Do we expect manual editions on these files?

Reply:
I'm being (probably overly) cautious with NDJSON, as we've learned the hard way with Kibana import/export that making the format evolve can be a pain (very difficult to add global metadata in addition to the entries). But for Kibana import/export, we're only generating a single file, we don't have a full folder structure like we do there, so the limitations of the format were more impactful. 

So no, I don't foresee us needing to add anything to those files, so if you prefer using ndjson, I'm fine with it

Reply:
Not a strong opinion, but I think I'd prefer using ndjson here, yes. If there is some case were we need to add metadata in the future we can use the manifest file, or some other file.